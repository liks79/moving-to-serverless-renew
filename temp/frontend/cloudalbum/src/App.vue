<template>
  <v-app>
    <layout-header></layout-header>
    <layout-body></layout-body>
    <layout-footer></layout-footer>
  </v-app>
</template>

<script>
import LayoutHeader from './components/layout/LayoutHeader.vue';
import LayoutBody from './components/layout/LayoutBody.vue';
import LayoutFooter from './components/layout/LayoutFooter.vue';

export default {
  name: 'App',
  components: { LayoutFooter, LayoutBody, LayoutHeader },
  data: () => ({
    //
  }),
};
</script>
