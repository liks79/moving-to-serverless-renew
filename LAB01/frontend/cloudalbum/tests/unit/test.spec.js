import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App', () => {
  const wrapper = shallowMount(App);

  it('renders the component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('renders layout-header', () => {
    expect(wrapper.find('layout-header-stub').exists()).toBe(true);
  });

  it('renders layout-body', () => {
    expect(wrapper.find('layout-body-stub').exists()).toBe(true);
  });

  it('renders layout-footer', () => {
    expect(wrapper.find('layout-footer-stub').exists()).toBe(true);
  });
});
