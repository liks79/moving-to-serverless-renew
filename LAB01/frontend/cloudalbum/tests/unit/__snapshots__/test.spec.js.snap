// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the component 1`] = `
<v-app-stub id="app">
  <layout-header-stub></layout-header-stub>
  <layout-body-stub></layout-body-stub>
  <layout-footer-stub></layout-footer-stub>
</v-app-stub>
`;
